web/src/components/ProtectedRoute.js

```diff
@@ -13,16 +14,19 @@ class ProtectedRoute extends React.PureComponent {
         const { setKeycloakSession } = this.props;
         const keycloak = await Keycloak("/keycloak.json");
 
-        keycloak.init({onLoad: 'login-required'}).success(authenticated => {
+        keycloak.init({onLoad: 'login-required', checkLoginIframe: false}).success(authenticated => {
```

Comment:
@moul This was necessary due to a keycloak issue with Chrome since version 80 released last month, now status iframe will only be able to see the special cookie over the SSL / TLS connection configured on the project side. You can check it here: https://github.com/keycloak/keycloak-documentation/blob/master/securing_apps/topics/oidc/javascript-adapter.adoc#session-status-iframe

Reply:
ok :+1: